web/controllers/organization_controller.ex

```diff
@@ -0,0 +1,55 @@
+defmodule CodeCorps.OrganizationController do
+  use CodeCorps.Web, :controller
+
+  alias CodeCorps.Organization
+  alias JaSerializer.Params
+
+  plug :scrub_params, "data" when action in [:create, :update]
+
+  def index(conn, _params) do
+    organizations = Repo.all(Organization)
+    organizations = Repo.preload(organizations, [:slugged_route])
```

Comment:
Just a suggestion here, but I'm a fan of creating composable queries in the model class. 

Something like this: http://blog.drewolson.org/composable-queries-ecto/